src/sys/node/public.ts

```diff
@@ -3,19 +3,25 @@ import type { CompilerSystem, Logger } from '@stencil/core/internal';
 /**
  * Creates a "logger", based off of NodeJS APIs, that will be used by the compiler and dev-server.
  * The NodeJS "process" object must be provided as a property in the first argument's object.
- * @param c an object containing a `Process` for Stencil to use
  * @returns a {@link Logger} object
  */
-export declare function createNodeLogger(c: { process: any }): Logger;
+export declare function createNodeLogger(): Logger;
```

Comment:
It looks like we removed this in https://github.com/ionic-team/stencil/commit/c114f0a9ce4352672d86a8605df53f8b7d81aa4f - I don't see any comments to this effect, but I _think_ we held off here due to breaking a public API (`createNodeLogger` is a available from `@stencil/core/sys/node`). @alicewriteswrongs do you remember by chance?

Reply:
yeah I think that might be why we left it like that, I don't remember exactly tbh

Reply:
I created two versions and deprecated one which will result in the following:
<img width="312" alt="Screenshot 2024-02-21 at 7 49 19 AM" src="https://github.com/ionic-team/stencil/assets/731337/e36b5851-548b-41e1-ae3c-a533926b8040">